### PR TITLE
:bug: Change Preemption column tooltip to indicate that tasks are re-scheduled

### PR DESCRIPTION
**Pull request by rszwajko**
Resolves: #2053

### CODE REVIEW

**Comment by rszwajko** [comment]
image

**Review by sjd78 — APPROVED** [review]
LGTM

@jortel - Does this version of the tooltip text look correct to you?

File: client/public/locales/en/translation.json [hunk 1]
```diff
@@ -526,7 +526,7 @@
   },
   "tooltip": {
     "priority": "Tasks priority, a non-negative number, impacts tasks scheduling policy. Lowest priority is 0. Higher priority tasks run before lower priority tasks.",
-    "preemption": "If enabled, allows the scheduler to cancel a running task and free the resources for higher priority tasks."
+    "preemption": "If enabled, allows the scheduler to re-schedule a running task and free the resources for higher priority tasks."
```
Comment (jortel) [inline client/public/locales/en/translation.json:529]: recommend: 
"preemption": "If enabled, allows the scheduler to reschedule other running tasks to free cluster resources needed to run higher priority tasks."

Comment (rszwajko) [reply client/public/locales/en/translation.json:529]: @jortel 
> reschedule other running tasks
>
This part is not clear for me. Does it mean that preemption flag set on task A affects scheduling of all other tasks (but not directly task A) ?

Comment (jortel) [reply client/public/locales/en/translation.json:529]: Yes. exactly.

**Pull request merged**